capnp/Cargo.toml

```diff
@@ -25,7 +25,7 @@ quickcheck = { version = "0.9", optional = true }
 quickcheck = "0.9"
 
 [features]
-default = ["std"]
+default = ["std", "sync_reader"]
```

Comment:
I would prefer to make the new feature not a default feature, so that we minimize the chance of breaking anyone's code.